api-doc.yaml

```diff
@@ -0,0 +1,1633 @@
+      required: true
+      schema:
+        type: string
+        format: uuid
+    ReleaseIdParam:
+      name: release-id
+      in: path
+      description: Release ID
+      required: true
+      schema:
+        type: string
+        format: uuid
+    AttachmentIdParam:
+      name: attachment-id
+      in: path
+      description: Attachment ID
+      required: true
+      schema:
+        type: string
+        format: uuid
+  schemas:
+    UnprocesssableEntityError:
+      description: 'Validation errors'
+      type: object
+      properties:
+        error:
+          type: object
+          properties:
+            field1:
+              type: string
+              example: "Validation error"
+            field2:
+              type: string
+              example: "Validation error"
+            object1:
+              type: object
+              properties:
+                field1:
+                  type: string
+                  example: "Validation error"
+      required:
+        - error
+    BadRequestError:
+      type: object
+      properties:
+        error:
+          type: string
+          example: "The request is malformed or contains invalid syntax."
+    NotFoundError:
+      type: object
+      properties:
+        error:
+          type: string
+          example:  'Resource not found.'
+    UnauthorizedError:
+      type: object
+      properties:
+        error:
+          type: string
+          example: 'JWT token is missing or invalid.'
+    UnauthorizedErrorGitRepository:
+      type: object
+      properties:
+        error:
+          type: string
+          example: 'Authentication to https://github.com/organization/repo failed. Organization token for GitHub API is missing.'
+    ForbiddenError:
+      type: object
+      properties:
+        error:
+          type: string
+          example:  'You do not have permission to perform this action.'
+    UserResponse:
+      type: object
+      properties:
+        id:
+          type: string
+          format: uuid
+        email:
+          type: string
+          format: email
+          example: jan@zabloudil.cz
+        name:
+          type: string
+          example: 'Jan Zabloudil'
+        avatar_image_url:
+          type: string
+        is_admin:
+          type: boolean
+          example: false
+        created_at:
+          type: string
+          format: date-time
+        updated_at:
+          type: string
+          format: date-time
+      required:
+        - id
+        - email
+        - name
+        - avatar_image_url
+        - is_admin
+        - created_at
+        - updated_at
+    ProjectRequest:
+      type: object
+      properties:
+        name:
+          type: string
+          example: 'Awesome project'
+        description:
+          type: string
+          example: 'This project is about...'
+        notifications:
+          type: object
+          properties:
+            slack_channel_id:
+              type: string
```

Comment:
Originally, users were able to submit an array of Slack channel names. However, I made the following changes:
* Slack channel ID instead of the name, as it serves as a better identifier.
* Only one Slack channel can be provided. Since STRV typically has one channel per project, it is sufficient for the MVP and will streamline implementation.